### PR TITLE
Reports documentation update

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

### :nut_and_bolt: Description: What code changed, and why?

The documentation for _Reports_ in the Compliance Dashboard needs updating. This PR follows the correct file structure for the new Chef Automate docs, unlike https://github.com/chef/automate/pull/4143.

### :chains: Related Resources

Original PR - https://github.com/chef/automate/pull/4143

This PR hopes to address some of the following issues with its update:
-  https://github.com/chef/automate/issues/4024
- https://github.com/chef/release-engineering/issues/1185
- https://github.com/chef/automate/issues/2534

### :+1: Definition of Done

🟢  means go

### :athletic_shoe: How to Build and Test the Change

`make serve`

Go to localhost:1313/docs/reports/

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/docs-chef-io/content/automate/reports.md [hunk 1]
```diff
@@ -13,28 +13,26 @@ draft = false
...
+The control count in the _Profile Status_ pane (Reports > Profile Status) shows the total number of executed controls, not the total number of unique controls.
+{{% /info %}}
 
-![Compliance](/images/automate/compliance.png)
+![Reports](/images/docs/reports.png)
 
 ### Dates in Compliance Reports
 
 The dashboard shows the results of all scans with end times on the _currently selected day_.
 The currently selected day, as highlighted in the search bar, bases itself on timestamps in Coordinated Universal Time (UTC).
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:28]:
```suggestion
The _currently selected day_, as highlighted in the search bar, bases itself on timestamps in Coordinated Universal Time (UTC).
```

@vjeffrey or @susanev Is this a Unix epoch timestamp or something else?

File: components/docs-chef-io/content/automate/reports.md [hunk 2]
```diff
@@ -65,97 +63,94 @@ Recipe
 
 ### Deep Filtering
 
-Deep Filtering allows you to view the state of your infrastructure from the perspective of a single profile, or a single profile and one of its child controls.  
-Adding a profile filter will modify all compliance reports screens to display the status of that profile across all nodes executing that profile. Adding a filter for a child control of that profile will modify all compliance reports screens to display the status of that child control.
+Deep filtering shifts the perspective of the results from the node to the profile and its associated control.
+Use deep filtering to see the compliance reporting for an entire profile or an individual control within a profile.
 
 Please note the only supported filters for deep filtering are:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:69]:
```suggestion
Deep filtering supports filtering for:
```

File: components/docs-chef-io/content/automate/reports.md [hunk 3]
```diff
@@ -65,97 +63,94 @@ Recipe
...
+    * one profile and one of its associated controls
 
 ### Waivers
 
-A node's waived status appears if applicable in displays where a node's status appears in Chef Automate. 
+A node's waived status appears if applicable in displays where a node's status appears in Chef Automate.
 The Compliance Reports overview displays the node count and history of waived nodes, and the count and history of waived controls.
-_Nodes_ and _Profiles_ views include _Waived Nodes_ and _Waived Profiles_ status filters respectively. 
+_Nodes_ and _Profiles_ views include _Waived Nodes_ and _Waived Profiles_ status filters respectively.
 Select the _Waived_ status filter to display only the respective Node or Profile reporting with that status.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:79]:
```suggestion
Selecting the _Waived_ status filter to displays the respective Node or Profile reporting with the `waived` status.
```

File: components/docs-chef-io/content/automate/reports.md [hunk 4]
```diff
@@ -65,97 +63,94 @@ Recipe
...
 
-![Compliance Nodes](/images/automate/compliance-nodes.png)
+![Reports Nodes](/images/docs/reports-nodes.png)
 
 Node
 : A node is any machine that is under management by Chef.
 
 Platform
-: The operating system on your node such as AIX, Amazon Linux, Apache, AppleOSX, CentOS, Oracle Linux, Oracle Solaris, RHEL, SUSE Linux, Ubuntu, and Microsoft Windows.
+: The operating system on your node, such as AIX, Amazon Linux, Apache, macOS, CentOS, Oracle Linux, Oracle Solaris, RHEL, SUSE Linux, Ubuntu, and Microsoft Windows.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:153]:
```suggestion
: The operating system on your node, such as AIX, Amazon Linux, Apache, macOS, CentOS, Oracle Linux, Oracle Solaris, RHEL, SUSE Linux, Ubuntu, and Windows.
```

I'll update the linter

File: components/docs-chef-io/content/automate/reports.md [hunk 5]
```diff
@@ -65,97 +63,94 @@ Recipe
...
+_Nodes_ and _Profiles_ views include _Waived Nodes_ and _Waived Profiles_ status filters respectively.
 Select the _Waived_ status filter to display only the respective Node or Profile reporting with that status.
 Hover over the control's Waived icon under the Node Status column in _Controls_ to view more details about the waiver applied to the control.
 
-Use Chef InSpec to configure [waivers](https://docs.chef.io/inspec/waivers/).
+Use Chef InSpec to configure [waivers](https://www.inspec.io/docs/reference/waivers/).
 
 ### Download Report Results
 
 The download button located to the right of the search bar allows the user to download a JSON or CSV format of the reports, based on all currently applied filters (including end time selected in calendar).
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:86]:
```suggestion
The download button located to the right of the search bar allows users to download a JSON or CSV format of the reports. The downloaded contents are the result of all of the applied filters--including end time selected in calendar.
```

File: components/docs-chef-io/content/automate/reports.md [hunk 6]
```diff
@@ -65,97 +63,94 @@ Recipe
...
+Expand the `Report Metadata` information by selecting the compliance status bar. The report metadata shows a summary of the nodes, report date, duration, status, number of platforms, number of environments, and number of profiles used in your scan.
 
-![Compliance Metadata](/images/automate/compliance-metadata.png)
+![Reports Metadata](/images/docs/reports-metadata.png)
 
-## Status Views
+## Compliance Overview
 
-The compliance overview provides insights into the status of your system. Toggle between the _Node Status_ and _System Status_ to view your system's overall compliance.
+Toggle between _Node Status_ and _Profile Status_ to view your system's overall compliance.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:97]:
```suggestion
Toggle between _Node Status_ and _Profile Status_ to view your system's compliance state.
```

File: components/docs-chef-io/content/automate/reports.md [hunk 7]
```diff
@@ -65,97 +63,94 @@ Recipe
...
+: Shows the severity of the compliance scan failures on your system's nodes.
 
-Global Compliance Over Time
-: Represents the changes in size and compliance status of your system over times. The calendar in the upper right corner of the visualization allows you to change the dates represented in this view,
+Node Status Over Time
+: Shows the changes in size and compliance status of your system over time. Use the dropdown menu in the upper left corner to change the date range. Changing the date range also changes the displays in related charts and tables.
 
 Top Platform Failures
-: Represents the amount of compliance scan failures ranked by operating system. Hovering over an individual bubble shows the platform and the number of nodes impacted.
+: Shows the number of compliance scan failures grouped by operating system. Hovering over an individual bubble shows the platform name and the number of impacted nodes.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:113]:
```suggestion
: Shows the number of compliance scan failures grouped by operating system. Hovering over an individual bubble shows the platform name and the number of failed nodes.
```

File: components/docs-chef-io/content/automate/reports.md [hunk 8]
```diff
@@ -65,97 +63,94 @@ Recipe
...
 
  Top Environment Failures
-: Represents the amount of compliance scan failures ranked by environment. Hovering over an individual bubble shows the environment and the number of nodes impacted.
+: Shows the number of compliance scan failures grouped by environment. Hovering over an individual bubble shows the environment name and the number of impacted nodes.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:116]:
```suggestion
: Shows the number of compliance scan failures grouped by environment. Hovering over an individual bubble shows the environment name and the number of failed nodes.
```

File: components/docs-chef-io/content/automate/reports.md [hunk 9]
```diff
@@ -65,97 +63,94 @@ Recipe
...
+: Shows failed controls, ranked by number and importance of the control failure, grouped as critical, major, and minor.
 
-Test Results Over Time
-: A line graph representing the quantity of controls and Compliance scan results over time. The time-frame can be altered using calendar button in the upper right corner of the chart.
+Control Status Over Time
+: A line graph showing the number of controls and Compliance scan results over time. Use the calendar button in the upper right corner of the chart to change the time frame.
 
 Top Profile Failures
-: Indicates the profiles with the highest failure rate. Hovering over an individual bubble shows the full name of the profile and the number of nodes impacted.
+: Indicates the profiles with the highest failure rate. Hover over an individual bubble to show the full profile name and the number of impacted nodes.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:132]:
```suggestion
: Indicates the profiles with the highest failure rate. Hover over an individual bubble to show the full profile name and the number of failed nodes.
```

File: components/docs-chef-io/content/automate/reports.md [hunk 10]
```diff
@@ -65,97 +63,94 @@ Recipe
...
 
 Top Control Failures
-: The Top Control Failures chart illustrates the most frequently failing controls. Hovering over an individual bubble shows the name of the control and the number of nodes impacted.
+: Indicates the most frequently failing controls. Hover over an individual bubble to show the control name and the number of impacted nodes.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/reports.md:135]:
```suggestion
: Indicates the most frequently failing controls. Hover over an individual bubble to show the control name and the number of failed nodes.
```

File: components/docs-chef-io/content/automate/reports.md (hunk 3 above)
Comment (mjingle) [reply components/docs-chef-io/content/automate/reports.md:79]: Fixed it: 
> Selecting the _Waived_ status filter displays the respective Node or Profile reporting with the `waived` status.

File: components/docs-chef-io/content/automate/reports.md (hunk 1 above)
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/reports.md]: @mjingle use the existing language & make a card to follow up on the timestamps next week.

**Comment by kagarmoe** [comment]
The images aren't showing up in Netlify.

File: components/docs-chef-io/content/automate/reports.md (hunk 1 above)
Comment (mjingle) [reply components/docs-chef-io/content/automate/reports.md]: card made

File: components/docs-chef-io/content/automate/reports.md [hunk 11]
```diff
@@ -65,97 +63,95 @@ Recipe
...
+A node's waived status appears if applicable in displays where a node's status appears in Chef Automate.
 The Compliance Reports overview displays the node count and history of waived nodes, and the count and history of waived controls.
-_Nodes_ and _Profiles_ views include _Waived Nodes_ and _Waived Profiles_ status filters respectively. 
-Select the _Waived_ status filter to display only the respective Node or Profile reporting with that status.
+_Nodes_ and _Profiles_ views include _Waived Nodes_ and _Waived Profiles_ status filters respectively.
+Selecting the _Waived_ status filter displays the respective Node or Profile reporting with the `waived` status.
 Hover over the control's Waived icon under the Node Status column in _Controls_ to view more details about the waiver applied to the control.
 
-Use Chef InSpec to configure [waivers](https://docs.chef.io/inspec/waivers/).
+Use Chef InSpec to configure [waivers](https://www.inspec.io/docs/reference/waivers/).
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/reports.md:82]: The original link is correct here.
```suggestion
Use Chef InSpec to configure [waivers](https://docs.chef.io/inspec/waivers/).
```

File: components/docs-chef-io/content/automate/reports.md (hunk 1 above)
Comment (mjingle) [reply components/docs-chef-io/content/automate/reports.md]: Received confirmation from VJ that _current selected day_ is a RFC3339 formatted timestamp.

File: components/docs-chef-io/content/automate/reports.md (hunk 11 above)
Comment (mjingle) [reply components/docs-chef-io/content/automate/reports.md]: Thank you for catching that, Ian. I checked and the rest of the links should be okay.

**Pull request merged**